text/storage-maps.md

```diff
@@ -91,5 +91,5 @@ let entry = <SimpleMap<T>>::take(&user);
 
 The rest of the API is documented in the rustdocs on the
 [`StorageMap` trait](https://substrate.dev/rustdocs/v2.0.0/frame_support/storage/trait.StorageMap.html). You do
-not need to explicitly `use` this trait because the `decl_storage!` macro will do it for you if you
+not need to explicitly use this trait because the `decl_storage!` macro will do it for you if you
```

Comment:
I would prefer to keep "use" formatted as code _or_ replace the word "use" with the word "import".